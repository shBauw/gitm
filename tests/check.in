place J11
place A1
place I10
place S1
place K10
place A10
place J9
place S10
place I9
place A2
place K9
place S2
place I11
place A9
place K11
place S6
place J10
view
term
