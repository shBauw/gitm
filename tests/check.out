C18,xxxxxxxxxxxxxxx......x......x......x......x......
